app/src/ui/lib/context-menu.ts

```diff
@@ -1,4 +1,6 @@
 const RestrictedFileExtensions = ['.cmd', '.exe', '.bat', '.sh']
+export const CopyFilePathLabel = 'Copy File Path'
```

Comment:
Could we please conditionalize this for Windows? macOS expects Title Case but for Windows it should be Sentence case.

```suggestion
export const CopyFilePathLabel = __DARWIN__ ? 'Copy File Path' : 'Copy file path' 
```